packages/server/graphql/public/typeDefs/OAuth.graphql

```diff
@@ -0,0 +1,113 @@
+extend type Organization {
```

Comment:
-1 i think this'll be the first instance where we use `extend type` instead of modifying the `Organization.graphql` or `Mutation.graphql` files directly. We used to do it this way! Unforrtunately we had types override types & people didn't see types & it caused subtle errors that went undetected in code reviews. If we can, it's great to keep them all together.
That said, tooling may be better now? We could have that discussion though if we want to break convention here.

The convention is: each type gets its own file